packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsForm.tsx

```diff
@@ -1,75 +1,58 @@
 import React from 'react';
 import {
-  FieldValues,
   Grid,
   Item,
-  ModalInput,
+  BasicTextInput,
   ModalLabel,
   ModalRow,
-  UseFormRegister,
   Select,
   useTranslation,
   InputLabel,
-  ModalNumericInput,
+  NumericTextInput,
   Divider,
   Box,
   Typography,
 } from '@openmsupply-client/common';
-import { ItemSearchInput } from '@openmsupply-client/system/src/Item';
-import { OutboundShipment, OutboundShipmentSummaryItem } from '../../../types';
-import { PackSizeController } from './ItemDetailsModal';
+import { ItemSearchInput } from '@openmsupply-client/system';
+import { PackSizeController } from './hooks';
+import { useOutboundRows } from '../../api';
 
 interface ItemDetailsFormProps {
   allocatedQuantity: number;
-  register: UseFormRegister<FieldValues>;
-  summaryItem?: OutboundShipmentSummaryItem;
+  availableQuantity: number;
+  item: Item | null;
   onChangeItem: (newItem: Item | null) => void;
   onChangeQuantity: (quantity: number, packSize: number | null) => void;
   packSizeController: PackSizeController;
-  availableQuantity: number;
-  draft: OutboundShipment;
 }
 
 export const ItemDetailsForm: React.FC<ItemDetailsFormProps> = ({
   allocatedQuantity,
   onChangeItem,
   onChangeQuantity,
-  register,
-  summaryItem,
+  item,
   packSizeController,
   availableQuantity,
-  draft,
 }) => {
   const t = useTranslation(['distribution', 'common']);
   const quantity =
-    allocatedQuantity / Math.abs(Number(packSizeController.selected.value));
+    allocatedQuantity /
+    Math.abs(Number(packSizeController.selected.value || 1));
```

Comment:
I had a thing with adding new items that had zero stock - the placeholder is the only line, which wasn't being counted in the pack size controller value, so was never showing